Unexpected error after trying to use a configuration file

In the latest commit on https://github.com/ericcornelissen/shescape/pull/1883, I updated `dirty-waters-action` to v1.11.43 and tried to add a configuration file. From what I could quickly tell (though I didn't spend too much time debugging yet) it looks correct to me but I'm getting a warning that the config file is not found followed by some (related?) error. Here's the (trailing) logs from GitHub Actions:

```log
Running command: python main.py -p ericcornelissen/shescape -v HEAD -pm npm -n --check-code-signature --check-source-code --check-source-code-sha --config ./config/dirty-waters.json --gradual-report=false
2025-04-23 11:43:06,407:root:WARNING:Config file not found at ./config/dirty-waters.json, using default config
Creating new table: github_urls
Creating new table: pr_info
Creating new table: pr_reviews
Creating new table: tag_to_sha
Creating new table: package_analysis
Creating new table: commit_authors_from_tags
Creating new table: commit_authors_from_url
Creating new table: patch_authors_from_sha
Creating new table: user_commit
Creating new table: extracted_dependencies
Traceback (most recent call last):
File "/home/runner/work/shescape/shescape/dirty-waters/tool/main.py", line 640, in <module>
main()
~~~~^^
File "/home/runner/work/shescape/shescape/dirty-waters/tool/main.py", line 617, in main
resolve_config_conflicts(project_info["config"])
~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^
File "/home/runner/work/shescape/shescape/dirty-waters/tool/main.py", line 456, in resolve_config_conflicts
for package, checks in checks_per_type.items():
^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'list' object has no attribute 'items'
Error: Process completed with exit code 1. 
```

--- a/action.yml
+++ b/action.yml
@@ -163,8 +163,8 @@ runs:
         fi
 
         if [ -n "${{ inputs.config }}" ]; then
-            cp $GITHUB_WORKSPACE/${{ inputs.config }} .
-            CMD="$CMD --config ${{ inputs.config }}"
+            #cp $GITHUB_WORKSPACE/${{ inputs.config }} .
+            CMD="$CMD --config $GITHUB_WORKSPACE/${{ inputs.config }}"
         fi
 
         if [ "${{ inputs.gradual_report }}" == "false" ]; then

--- a/action.yml
+++ b/action.yml
@@ -121,7 +121,7 @@ runs:
         git clone https://github.com/chains-project/dirty-waters.git
         cd dirty-waters
         # The version is pinned to the latest dirty-waters release at the time of the action release
-        DIRTY_WATERS_VERSION="v0.95.0"
+        DIRTY_WATERS_VERSION="v0.96.0"
         git checkout $DIRTY_WATERS_VERSION
         pip install -r requirements.txt
 

--- a/action.yml
+++ b/action.yml
@@ -121,7 +121,7 @@ runs:
         git clone https://github.com/chains-project/dirty-waters.git
         cd dirty-waters
         # The version is pinned to the latest dirty-waters release at the time of the action release
-        DIRTY_WATERS_VERSION="v0.96.0"
+        DIRTY_WATERS_VERSION="v0.97.0"
         git checkout $DIRTY_WATERS_VERSION
         pip install -r requirements.txt
 

